src/components/TextBubble/TextBubbleService.tsx

```
@@ -0,0 +1,84 @@
+              <div className="border-t border-gray-300 my-3" />
+              <div
+                className="text-gray-400 whitespace-pre-wrap"
+                dangerouslySetInnerHTML={{ __html: MOCK_SIMPLIFIED }}
+              />
+            </>
+          )}
+        </div>
+
+        {/* 번역 버튼 */}
+        <button
+          onClick={() => setShowSimplified((prev) => !prev)}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)